migrations/migrations/20181023175544-users-model.js

```diff
@@ -32,6 +32,9 @@ module.exports = {
         defaultValue: false,
         allowNull: false
       },
+      sessionInvalidate: {
```

Comment:
Remember the idea of migrations is to add a new migration each time you make some change in the db schema